drivers/filesystems/fastfat/vfat.h

```diff
@@ -1221,6 +1221,10 @@ BOOLEAN
 vfatIsLongIllegal(
     WCHAR c);
 
+BOOLEAN
+IsDotOrDotDot(
```

Comment:
Would it be worth to be inline or even a macro?

Reply:
@SergeGautherie, I followed the same pattern as the function above which is "vfatIsLongIllegal". This makes it consistent with what was done previously. If any of the other developers believes that this is beneficial, then maybe it can be added later as a follow-up to this PR. My preference is not to make changes to something that already has two approvals. Thanks.